### PR TITLE
chore: Add vercel & sentry as sponsors

### PR DESCRIPTION
**Pull request by dwelle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,10 @@ If you like the project, you can become a sponsor at [Open Collective](https://o
 
 <a href="https://opencollective.com/excalidraw#category-CONTRIBUTE" target="_blank"><img src="https://opencollective.com/excalidraw/tiers/backers.svg?avatarHeight=32"/></a>
 
+Last but not least, these companies are sponsoring Excalidraw by offering their services:
+
+[![Vercel](./images/vercel.svg)](https://vercel.com) [![Sentry](./images/sentry.svg)](https://sentry.io)
```
Comment (lipis) [inline README.md:33]:
```suggestion
[![Vercel](images/vercel.svg)](https://vercel.com) [![Sentry](images/sentry.svg)](https://sentry.io)
```

File: images/vercel.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+</svg>
```
Comment (lipis) [inline images/vercel.svg:3]:
```
svgo --pretty --indent=2
```

File: README.md (hunk 1 above)
Comment (lipis) [reply README.md:33]: maybe `assets` to be more generic for future stuff?

Comment (lipis) [reply README.md:33]: I think we can add images under `.github` since it's part of the GitHub readme front page :)

File: images/vercel.svg (hunk 2 above)
Comment (dwelle) [reply images/vercel.svg:3]: When I tried SVGO (web version) on this image it completely broke it :(

Comment (dwelle) [reply images/vercel.svg]: actually, vercel was piped through SVGO

**Comment by dwelle** [comment]
Let's merge this and we can improve it later.

**Pull request merged**